.github/workflows/ci_vtests.yml

```diff
@@ -17,51 +17,33 @@ jobs:
       if: github.event_name == 'pull_request'
       run: |
         export BASEREF=${{ github.event.pull_request.base.sha }}
-        if [ -z "$BASEREF" ]; then
-        export found=0
-        else
-        export found=1
-        fi
+        if [ -z "$BASEREF" ]; then found=0; else found=1; fi
         echo "BASEREF=$BASEREF" >> $GITHUB_ENV
         echo "found=$found" >> $GITHUB_ENV
     - name: Retrieve base commit for push commit
       if: github.event_name == 'push'
       run: |
         export BASEREF=$( git show -s --pretty=%P ${{ github.sha }} | head -c 10 )
-        if [ -z "$BASEREF" ]; then
-        export found=0
-        else
-        export found=1
-        fi
+        if [ -z "$BASEREF" ]; then found=0; else found=1; fi
         echo "BASEREF=$BASEREF" >> $GITHUB_ENV
         echo "found=$found" >> $GITHUB_ENV
-    - name: Install dependencies
+    - name: Setup environment
       if: contains( env.found, '1')
       run: |
-        sudo apt-get update
-        sudo apt-get install libasound2-dev portaudio19-dev libmp3lame-dev libsndfile1-dev libportmidi-dev
-        sudo apt-get install libssl-dev libpulse-dev libfreetype6-dev libfreetype6
-        sudo apt-get install libdrm-dev libgl1-mesa-dev libegl1-mesa-dev
-    - name: Setup the environment
-      if: contains( env.found, '1')
-      run: |
-        sed -i 's/travis_retry//g' build/travis/job1_Tests/environment.sh
-        source build/travis/job1_Tests/environment.sh
-        echo "PATH=$PATH" >> $GITHUB_ENV
-        echo "QT_PLUGIN_PATH=$QT_PLUGIN_PATH" >> $GITHUB_ENV
-        echo "QML2_IMPORT_PATH=$QML2_IMPORT_PATH" >> $GITHUB_ENV
-        git checkout -- build/travis/job1_Tests/environment.sh
+        sudo bash ./build/ci/linux/setup.sh
     - name: Build
       if: contains( env.found, '1')
       run: |
-        make installdebug CPUS=2 PREFIX="$HOME/software" COVERAGE=ON
+        . ./../musescore_environment.sh
```

Comment:
the `.` command is pretty easy to get overlooked, maybe better use the `source` command?

Reply:
I agree. Changed.

Reply:
This `. ./../` looks like ancient Cuneiform ;-)